src/translations/nbTranslations.json

```diff
@@ -1275,7 +1275,7 @@
       "handle": "Handle",
       "in_progress": "Under behandling",
       "message_to_curator": "Melding til kurator",
-      "metadata_not_published": "Metadata er ikke publisert",
+      "metadata_not_published": "Resultatet er ikke publisert",
```

Comment:
Fortsatt riktig at denne heter "**metadata**_not_published"?

Reply:
var egentlig bevisst ja, men når du sier det så burde jeg kanskje bare rename også. Litt sånn house-keeping man får betalt senere for å gjøre nå 😅 oppdaterer jeg